test/hotspot/jtreg/TEST.groups

```diff
@@ -599,6 +601,7 @@ hotspot_tier2_runtime = \
  -runtime/CommandLine/OptionsValidation/TestOptionsWithRanges.java \
  -runtime/CompressedOops/UseCompressedOops.java \
  -runtime/InvocationTests \
+ -runtime/Monitor/StressWrapper_TestRecursiveLocking_36M.java \
```

Comment:
Does runtime/Monitor/TestRecursiveLocking.java need to be added in here too?

Reply:
OH I see there's a minus sign here too for tier2, so I assume it'll run in tier3?  But the 5 second one will run in tier2.

Reply:
Yes. Taken from Dan's original PR: _"The default version of the test is NOT run in Tier1; it is run in Tier2 intentionally since this is a stress test."_  and  _"The StressWrapper_TestRecursiveLocking_36M.java version of the test is run in Tier3 and is NOT run in Tier1 or Tier2."_